node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -210,12 +215,15 @@ impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
 
     // Returns number of elements in `vertex_deps` queue.
     fn vertex_deps_len(&self) -> u64 {
```

Comment:
```suggestion
    fn vertices_awaiting_deps_len(&self) -> u64 {
```

Reply:
Changed in [`d3e8a82` (#1312)](https://github.com/CasperLabs/casper-node/pull/1312/commits/d3e8a82bae9fc19cf5a7839a640b0a219720e6c5)